shoop/core/models/_contacts.py

```diff
@@ -37,7 +39,7 @@ class Meta:
         verbose_name_plural = _('contact groups')
 
     def __str__(self):
-        return self.safe_translation_getter("name", default="Group<%s>" % (self.identifier or self.id))
+        return force_text(self.safe_translation_getter("name", default="Group<%s>" % (self.identifier or self.id)))
```

Comment:
Why would this need force_text? IMHO `__str__` should rather just be removed from here, since `TranslatableShoopModel` has a good default.